run_bot.sh

```
@@ -590,6 +590,151 @@ test_recording() {
+    fi
+    
+    # Check for API request handling
+    if grep -q "end meeting from api server" "$log_file"; then
+        print_success "✅ API request received and handled"
+    else
+        print_warning "⚠️ No API request handling log found"
+    fi
+    
+    # Check for generated files
+    local bot_files_dir="$output_dir/$bot_uuid"
+    if [ -d "$bot_files_dir" ] && [ "$(find $bot_files_dir -name "*.mp4" -o -name "*.wav" | wc -l)" -gt 0 ]; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)